libcontainer/factory_nabla.go

```diff
@@ -64,7 +67,28 @@ type NablaFactory struct {
 	Root string
 }
 
-// TODO(NABLA)
+func createRootfsISO(config *configs.Config) (string, error) {
+	rootfsPath := config.Rootfs
+	isoPath, err := storage.CreateIso(rootfsPath)
+	if err != nil {
+		// TODO: Add wrap
+		return "", err
+	}
+	targetISOPath := filepath.Join(rootfsPath, "rootfs.iso")
+
+	if err = utils.Copy(targetISOPath, isoPath); err != nil {
+		// TODO: Do cleanup
+		return "", err
+	}
+
+	return targetISOPath, nil
+}
+
+// nablaTapName returns the tapname of a given container ID
+func nablaTapName(id string) string {
+	return ("tap" + id)[:syscall.IFNAMSIZ-1]
```

Comment:
Maybe check (or assert) that len(id) is at least of some size. Just to be sure that not all taps have the same name.

Reply:
Will add this!

Reply:
Addressed in b7e4d382f42657d2950ab630cb1d3582c0439571